be/src/column/array_column.cpp

```diff
@@ -384,6 +384,13 @@ void ArrayColumn::crc32_hash(uint32_t* hash, uint32_t from, uint32_t to) const {
     }
 }
 
+int64_t ArrayColumn::xor_checksum() const {
+    // The XOR of ArrayColumn
+    // XOR the offsets column and elements column
+    int64_t xor_checksum = _offsets->xor_checksum();
```

Comment:
offset record the position of items in chunk, if the chunk size is not consistency, the result is wrong.

Reply:
Yes, this is an error. I change it to XOR the size of every array.